actioncable/lib/action_cable.rb

```diff
@@ -37,6 +37,10 @@ module ActionCable
     },
     default_mount_path: '/cable'.freeze
   }
+  
+  # This protocol ought to be versioned separately
+  # from the rest of the Gem
+  PROTOCOLS = ["actioncable-protocol-1.0.0".freeze].freeze
```

Comment:
@matthewd This kind of gets to a bigger question of how much backwards-compatibility we build into this.

If somebody wants to make their ActionCable server a publicly available resource, then backwards compatibility becomes important. Likewise, I made a Swift client for ActionCable and a changing protocol version becomes a real pain for the developer because the longer upgrade cycle becomes a chicken-and-egg problem.

On the other hand, if this is really only used so a person can communicate with their own backend in the browser, there doesn't need to be a lot of backwards compatibility built-in. Only enough so that the client can gracefully degrade until the next reload.

In either case, there is no benefit for the developer in being able to pin a version because if the server is able to support a version, it should because it's an internal protocol.

I'm interested to see what feedback we get about this.